migrations/20160818182848_create_fda_approvals_and_add_relationship_with_documents.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+exports.up = (knex) => (
+    knex.schema
+      .createTable('fda_approvals', (table) => {
+        table.text('id').primary();
+
+        table.uuid('intervention_id')
+          .notNullable()
+          .references('interventions.id');
+        table.integer('supplement_number')
+          .notNullable();
+        table.text('type')
+          .notNullable()
+          .index();
+        table.date('action_date')
+          .notNullable();
+        table.text('notes')
+          .nullable();
+        table.timestamps();
+
+        table.unique(['intervention_id', 'supplement_number']);
+      })
+      .table('documents', (table) => {
+        table.text('fda_approval_id')
+          .nullable()
+          .references('fda_approvals.id');
+      })
+      .raw('ALTER TABLE documents ALTER COLUMN trial_id DROP NOT NULL')
+      .raw(`ALTER TABLE documents
+            ADD CONSTRAINT trial_id_xor_fda_approval_id_check CHECK (
```

Comment:
Other options is to have `entity_type` (`trial/fda_approval`) and `entity_id` instead of this a little bit weird situation with `trial_id/fda_approval_id`. Later documents could be linked to `conditions`, `organisations` etc.